src/content/tutorials/fleet-management/app-platform/app-bundle/index.md

```diff
@@ -22,80 +22,81 @@ last_review_date: 2024-10-28
+No, at least not in the current implementation form. The reason is that app bundles install `App` custom resources that are understood only by the management clusters. Attempting to install them directly to the workload cluster would result in an error. This is because the corresponding `App` custom resource definition isn't available there, which means Kubernetes cannot understand the submitted objects.
 </div>
 
-Now, obviously in order to leverage this management cluster layer, the bundle `App` custom resource must be configured in a certain way which is slightly different in comparison to the solitary apps. Find more details in the [Technical Details paragraph](#technical-details).
+Now, obviously to leverage this management cluster layer, the bundle `App` custom resource must be configured in a certain way which is slightly different compared to the solitary apps. Find more details in the [Technical Details paragraph](#technical-details).
 
 ### Technical details {#technical-details}
 
 #### Naming the bundle's `App` resource
 
-The first, not so obvious, consequence of installing a bundle in a management cluster before its nested apps are installed in the workload cluster, is the requirement for name uniqueness. When a given app bundle is to be installed for more than one workload cluster, each `App` custom resource instance of this bundle must be given a unique name within the management cluster.
+The first, not so obvious, consequence of installing a bundle in a management cluster before its nested apps are installed in the workload cluster is the requirement for name uniqueness. When a given app bundle is to be installed for more than one workload cluster, each `App` custom resource instance must be given a unique name within the management cluster.
 
-It becomes natural for solitary apps, for example, the [`Hello World` app](https://github.com/giantswarm/hello-world-app), that when one needs to install it twice in the workload cluster, both `App` custom resources instances must be named uniquely. It's natural because both live under the same namespace, but even if we could break them into separate namespaces, this uniqueness would still be demanded because both apps targets the same workload cluster, hence must be distinguished by its operators.
+It becomes natural for solitary apps, for example, the [`Hello World` app](https://github.com/giantswarm/hello-world-app), that when one needs to install it twice in the workload cluster, both `App` custom resource instances must be named uniquely. It's natural because both live under the same namespace. Even if we could break them into separate namespaces, this uniqueness would still be required because both apps target the same workload cluster and must be distinguished by its operators.
```

Comment:
```suggestion
It becomes natural for solitary apps (for example, the [Hello World app]) that when you need to install it twice in the workload cluster, both App custom resource instances must be named uniquely. It's natural because both live under the same namespace. Even if we could break them into separate namespaces, this uniqueness would still be required because both apps target the same workload cluster and must be distinguished by its operators.
```